keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -56,40 +56,37 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 
 ## Summary
 
-Add a new field `spec.internalTrafficPolicy` to Service that allows node-local routing for Service internal traffic.
+Add a new field `spec.trafficPolicy` to Service that allows node-local and topology-aware routing for Service traffic.
 
 ## Motivation
 
-Internal traffic routed to a Service is not topology aware today. The [Topolgoy Aware Subsetting](/keps/sig-network/2004-topology-aware-subsetting)
-KEP addresses topology aware routing for Services by subsetting endpoints to dedicated EndpointSlices.
-While this approach works for the standard zone/region topologies, it wouldn't work for node level
-topologies since that would require an EndpointSlice per node. In larger clusters this wouldn't scale well.
-
-This KEP proposes a new field in Service to treat node-local topologies as a first class concept in Service similar
-to `externalTrafficPolicy`. This addresses the node-local use-case for Service while avoiding EndpointSlice
-subsetting per node.
+Internal traffic routed to a Service has always been randomly distributed to all endpoints.
+This KEP proposes a new API in Service to address use-cases such as node-local and topology aware routing
+for internal Service traffic.
 
 ### Goals
 
-* Allow internal Service traffic to be routed to node-local endpoints.
+* Allow internal Service traffic to be routed to node-local or topology-aware endpoints.
 * Default behavior for internal Service traffic should not change.
 
 ### Non-Goals
 
-* Topology aware routing for zone/region topologies.
+* Topology aware routing for zone/region topologies -- while this field enables this feature, this KEP only covers node-local routing.
+  See the Topology Aware Hints KEP for more details.
 
 ## Proposal
 
-Introduce a new field in Service `spec.internalTrafficPolicy`. The field will have 3 codified values:
+Introduce a new field in Service `spec.trafficPolicy`. The field will have 4 codified values:
 1. Cluster (default): route to all cluster-wide endpoints (or use topology aware subsetting if enabled).
```

Comment:
should https://github.com/kubernetes/enhancements/pull/2166 be rebased or we could take the chance to update `Cluster` to `All`
cc @howardjohn